GameData/RealismOverhaul/Engine_Configs/NAA75_110_Config.cfg

```diff
@@ -270,7 +350,7 @@
 		ignitionReliabilityEnd = 0.97
 		cycleReliabilityStart = 0.77 // a bit higher than XLR43-NA-1
 		cycleReliabilityEnd = 0.94
-		techTransfer = XLR41-NA-1:30&XLR43-NA-1:50			// A-4/V-2 derivative. Modified XLR43-NA-1
+		techTransfer = XLR41-NA-1,XLR43-NA-1:50			// A-4/V-2 derivative. Modified XLR43-NA-1
```

Comment:
Why increase the transfer from the XLR41 here? (That opens up a larger question I guess, which I've just added to #2505)